app/assets/javascripts/utils/api.js

```diff
@@ -1,4 +1,5 @@
 import { capitalize } from './strings';
+import NotificationActions from '../actions/notification_actions.js';
```

Comment:
If we're going to couple this to the response from the server — which is a good idea, although not what I was imagining — then we should not doing this in API.js. As much as possible, API.js should be strictly about querying th endpoint and returning the data.

Adding the Notification could be done by adding handling for the NEEDS_UPDATE with the notification store. ServerActions.needsUpdate() already sends out the 'NEEDS_UPDATE' signal when this action succeeds. Following the patterns of this app would mean adding the 'NEEDS_UPDATE' case to the NotificationStore.